packages/guess-webpack/src/prefetch-aot-plugin.ts

```diff
@@ -110,15 +113,13 @@ export class PrefetchAotPlugin {
       fileChunk = res.fileChunk;
     } catch (e) {
       callback();
-      console.error(e);
+      this.logger.error(e);
```

Comment:
Also replaced all calls to `console` with the logger, not sure if you wanted this.